web/frontend/src/components/slam/index.svelte

```diff
@@ -117,23 +117,24 @@ const refresh2d = async () => {
       // assuming reconfigures do not happen at the nanosecond scale
       if (
         newLastReconfigured?.getSeconds() !== lastReconfigured?.getSeconds() ||
-        reloadMap === undefined
+        isLocalizingMode === undefined
       ) {
         lastReconfigured = newLastReconfigured;
 
         const props = await slamClient.getProperties();
-        reloadMap =
-          props.mappingMode !== slamApi.MappingMode.MAPPING_MODE_LOCALIZE_ONLY;
+        isLocalizingMode =
+          props.mappingMode === slamApi.MappingMode.MAPPING_MODE_LOCALIZE_ONLY;
       }
 
       /*
        * Update the map and pose if the SLAM session is in mapping/updating mode or the pointcloud
        * has yet to be defined else only update the pose
        */
-      if (reloadMap || pointcloud === undefined) {
+      if (!isLocalizingMode || pointcloud === undefined) {
         let response;
+        // only request the edited map if we are in localizing mode
```

Comment:
updating mode

Reply:
the behavior in this PR is setup to only return the edited map if we are in localizing mode. I thought that was what we had agreed on while we wait for the actual map editing to be implemented. If thats not the case, I can update this PR to return edited maps in updating mode instead of localizing mode

Reply:
I'm confused by the comment above this, then. Isn't this section of the code handling updating mode?

We're aligned on what the expected behaviour is.

Reply:
I think I'm just getting confused by comment placement int he code. What you did was correct

Reply:
sorry! all me

Reply:
I see. Basically for localizing mode we only request the pointcloud once, when the pointcloud is undefined. For mapping and updating mode we continue to request the pointcloud

Reply:
Oooooh, I see now.

Reply:
Thank you for double-checking with me regardless.